packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 22.5.1
```

Comment:
probably doesn't need to be a new version, could do NEXT instead. Not sure if it really matters though

Reply:
Since this is very clearly a no-op change other than analysis, it doesn't even need a NEXT entry; this falls under the "unless they only affect development of the package" exemption for CHANGELOG changes.

Reply:
How do I proceed then? Only removing those mentions for the new version from the branch or do I need something else?

Reply:
I'm not sure I understand the question. The changes in this PR don't require any version or CHANGELOG changes in `pigeon`.

Reply:
My question was if I need to do anything else then, to signify that this falls under the exemption for CHANGELOG changes. From your answer, I guess I don't. If I do, please say so, since this is my first contribution here and I'm not aware. Fixed the version changes. Thanks a lot!

Reply:
in this case I think no changelog or version change is the appropriate action

Reply:
nvm i see this was merged.